docs/domains/blas/blas-level-1-routines.rst

```diff
@@ -0,0 +1,76 @@
+.. _blas-level-1-routines:
+
+BLAS Level 1 Routines
+=====================
+
+
+.. container::
+
+
+   BLAS Level 1 includes routines which perform
+   vector-vector operations as described in the following table. 
+
+
+   .. container:: tablenoborder
+
+
+      .. list-table:: 
+         :header-rows: 1
+
+         * -     Routines
+           -     Description     
+         * -     \ `asum <asum.html>`__\   
+           -     Sum of vector magnitudes      
+         * -     \ `axpy <axpy.html>`__\   
+           -     Scalar-vector product      
+         * -     \ `copy <copy.html>`__\   
+           -     Copy vector      
+         * -     \ `dot <dot.html>`__\   
+           -     Dot product      
+         * -     \ `sdsdot <sdsdot.html>`__\   
+           -     Dot product with double precision      
+         * -     \ `dotc <dotc.html>`__\   
+           -     Dot product conjugated      
+         * -     \ `dotu <dotu.html>`__\
+           -     Dot product unconjugated      
+         * -     \ `nrm2 <nrm2.html>`__\   
+           -     Vector 2-norm (Euclidean norm)      
+         * -     \ `rot <rot.html>`__\
+           -     Plane rotation of points      
+         * -     \ `rotg <rotg.html>`__\   
+           -     Generate Givens rotation of points      
+         * -     \ `rotm <rotm.html>`__\   
+           -     Modified Givens plane rotation of points           
+         * -     \ `rotmg <rotmg.html>`__\   
+           -     Generate modified Givens plane rotation of points           
+         * -     \ `scal <scal.html>`__\
+           -     Vector-scalar product      
+         * -     \ `swap <swap.html>`__\   
+           -     Vector-vector swap      
+         * -     \ `iamax <iamax.html>`__\   
+           -     Index of the maximum absolute value element of a       vector     
```

Comment:
There's extra space here (between "a" and "vector") and in other places in the routine list files. But if it doesn't show up in the formatted output, feel free to ignore this comment :)

Reply:
I think, these came from the translator. I have been removing the ones I notice. I can remove this one as well :)